lib/assets/SnapshotHelper.swift

```diff
@@ -91,4 +105,4 @@ extension XCUIElement {
 
 // Please don't remove the lines below
 // They are used to detect outdated configuration files
-// SnapshotHelperVersion [[1.0]]
+// SnapshotHelperVersion [[1.1]]
```

Comment:
Nice! :+1: 

@KrauseFx what is the protocol for handling updates in the helper?